temporalio/worker/_activity.py

```diff
@@ -201,6 +201,7 @@ async def drain_poll_queue(self) -> None:
 
     # Only call this after run()/drain_poll_queue() have returned. This will not
     # raise an exception.
+    # TODO(dan): check accuracy of this comment; I would say it *does* raise an exception.
```

Comment:
@cretz can you help resolve this TODO / question?

Reply:
I think it's _probably_ a _mostly_ true statement because the `_run_activity` is written to hopefully trap exceptions, but I think the intention was to set `return_exceptions=True` instead of `False` below, so that's probably what we should do here.

Reply:
Thanks, yes that's exactly what I was thinking. Updated TODO comment to that effect and leaving it in.

Reply:
I would be totally ok changing the parameter

Reply:
OK, yes let's investigate that. But this is a pure refactoring PR and I don't like mixing functional changes with refactoring.